core/src/test/java/org/testcontainers/containers/ParsedDockerComposeFileValidationTest.java

```diff
@@ -18,7 +20,7 @@ public void shouldValidate() {
         File file = new File("src/test/resources/docker-compose-container-name-v1.yml");
         Assertions
             .assertThatThrownBy(() -> {
-                new ParsedDockerComposeFile(file);
+                new ParsedDockerComposeFile(file, emptyMap());
```

Comment:
Perhaps it would be worth having a constructor that omits the Map parameter, at least for testing purposes?

Reply:
I'm not sure. Having it as a parameter forces the user of the class to consider the env. vars (if any) to substitute with. A developer might not otherwise consider that parsing a compose file depends on environment variables.

But I can add a constructor variant defaulting to no env. vars if you disagree with this.